deploy/provider-vault.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: csi-secrets-store-provider-vault
+  name: csi-secrets-store-provider-vault
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store-provider-vault
+    spec:
+      tolerations:
+      containers:
+        - name: provider-vault-installer
+          image: hashicorp/secrets-store-csi-driver-provider-vault:0.0.1
```

Comment:
we could consider making the image tag configurable in values.yaml

Reply:
Updated the charts to make image tag and repo configurable.